Demos/ReactiveKafkaFsProducer/src/main/scala/ReactiveKafkaFsProducer.scala

```diff
@@ -25,19 +26,21 @@ object ReactiveKafkaFsProducer extends App {
   val MAX_ALLOWED_FILES     = 1000
   val config                = ConfigFactory.load()
   val dir                   = config.getOrElse("dataDir", "").toOption.fold("")(identity(_))
+
   val path                  = File(s"${dir}/Casalini_mrc").path.toString
 
   val bootstrapServers = config.getOrElse("bootstrapServers", "").toOption.fold("")(identity(_))
 
   val producerSettings = ProducerSettings(system, new StringSerializer, new ByteArraySerializer)
     .withBootstrapServers(bootstrapServers)
 
-  val source: Source[Path, NotUsed] = Directory.ls(FileSystems.getDefault.getPath(path))
+  val fs                            = FileSystems.getDefault
```

Comment:
AFAICT, the changes to this file are not semantic.  They are just backing out some changes I made in a previous commit, so I don't think you intend them to be part of this PR.  Can you exclude them?